Cart Flow/Custom Views/Cells/ItemListCell.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  ItemListCell.swift
+//  Cart Flow
+//
+//  Created by Scott Quintana on 8/5/20.
+//  Copyright © 2020 Scott Quintana. All rights reserved.
+//
+
+import UIKit
+
+protocol ItemListCellDelegate: class {
+    func didTapEditItemButton(for item: ShoppingItem)
+}
+
+class ItemListCell: UITableViewCell {
+
+    static let reuseID = "ItemListCell"
+    
+    let itemLabel = CFTitleLabel(textAlignment: .left, fontSize: 20)
+    let lastPurchasedLabel = UILabel()
+    let editItemButton = CFEditButton()
+    var shoppingItem: ShoppingItem? = nil
+    weak var delegate: ItemListCellDelegate!
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        configure()
+    }
+    
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    
+    func set(item: ShoppingItem) {
+        itemLabel.text = item.name
+        
+        let dateFormatter = DateFormatter()
```

Comment:
`DateFormatter`s are relatively expensive to instantiate. Instead of creating a new `DateFormatter` each time you set the `ShoppingItem` on a call here, consider moving the `DateFormatter` to a class property or even a static property so it can be instantiated once and used over and over again.